test/metabase/api/embed_test.clj

```diff
@@ -47,15 +47,15 @@
 (defmacro with-new-secret-key! {:style/indent 0} [& body]
   `(do-with-new-secret-key! (fn [] ~@body)))
 
-(defn card-token [card-or-id & [additional-token-params]]
-  (sign (merge {:resource {:question (u/the-id card-or-id)}
+(defn card-token [card-or-id & [additional-token-keys entity-id]]
```

Comment:
This is just a test function so consider this a very nitty nonblocking nit, but it feels a bit potentially confusing to ignore `card-or-id` totally when `entity-id` is passed. Not sure if it'd be better to have a different function that takes the `entity-id`, or what.

Reply:
Ahh, I think I should substitute u/the-id with

```
(defn the-id-or-entity-id [id-or-entity-id]
  (try (u/the-id id-or-entity-id)
       (catch Exception _ id-or-entity-id)))
```

and just pass the `entity-id`!

I'll make that change.

Reply:
yeah, this way 

`(card-query-url card response-format {} (:entity_id card))`

becomes

`(card-query-url (:entity_id card) response-format)`

🎉